docs/run.rst

```diff
@@ -143,3 +153,31 @@ Here is an example of a valid stage-in manifest file:
 ``/scratch/users/me/input_data/input_1.dat /unifyfs/input/input_1.dat``
 ``/home/users/me/configuration/run_12345.conf /unifyfs/config/run_12345.conf``
 ``"/home/users/me/file with space.dat" "/unifyfs/file with space.dat"``
+
+The transfer API stage functionality can also be used with a stand-alone
+application called "unifyfs-stage".  This function takes the argument
+of the name of a manifest file.  It can be run at any time, so could
+be run during a job, to transfer new into into the UnifyFS volume,
+or trasnfer results out and checked before the job ends.
+
+  Stand-alone file stage application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The manifest subsystem can also be used during the job by
+invoking stand-alone transfer executable.  The command to use is
+"transfer-static".  It's used similarly to the unix
+"cp" command, with source and destination, except that unify-static
+is aware that it is copying files between the external parallel
+file system and the internal UnifyFS volume.
+
+The stand-alone can be invoked at any time the UnifyFS servers
+are up and responding to requests.  Thus the user can move files
+in or out during a job where UnifyFS is running at any time
+during the job, if UnifyFS has been "start"ed as above.  This
+could be done during the job to bring in new input, or perhaps
+transfer results files out so they can be verified before they
+job terminates.
+
+Examples of use of the transfer-static command:
+srun -N 2 /path/to/libexec/transfer-static /path/on/parallelfs/file.dat /unifyfs/file.dat
```

Comment:
We may want to comment on how it is a parallel transfer, which helps to explain the '-N 2' arg to srun